packages/adblocker-benchmarks/package.json

```diff
@@ -14,19 +14,18 @@
     "directory": "packages/adblocker-benchmarks"
   },
   "scripts": {
-    "download-requests": "make requests.json",
-    "download-ubo": "make ubo-core",
-    "download-abp": "make adblockpluscore",
-    "download-tsurlfilter": "make tsurlfilter-node",
-    "download-brave": "make adblock-rs",
-    "download-fast-hosts-lookup": "make fast-hosts-lookup",
-    "download-deps": "yarn download-requests && yarn download-ubo && yarn download-abp && yarn download-tsurlfilter && yarn download-brave && yarn download-fast-hosts-lookup"
+    "postinstall": "make ubo-core"
   },
   "bugs": {
     "url": "https://github.com/ghostery/adblocker/issues"
   },
   "dependencies": {
+    "@adguard/tsurlfilter": "2.2.19",
     "@cliqz/adblocker": "^1.27.3",
+    "@eyeo/webext-ad-filtering-solution": "1.7.0",
+    "@gorhill/ubo-core": "https://github.com/gorhill/uBlock/releases/download/1.58.0/uBlock0_1.58.0.npm.tgz",
+    "adblock-rs": "0.8.10",
+    "fast-hosts-lookup": "https://github.com/mjethani/fast-hosts-lookup#9136bb8c331ee5dba8ac1a0018552cace11e2afa",
     "tldts-experimental": "^6.0.14"
```

Comment:
I think it's an issue to add them to the dependencies because now running `yarn bootstrap` at the top-level will install these dependencies (including compiling `adblock-rs` which can take a while). Ideally we would only install these dependencies lazily if we want to run the benchmarks.